### PR TITLE
Add cypress e2e tests

### PR DESCRIPTION
**Pull request by bidoubiwa**
TODO
- [x] remove angular own tester

Cypress is only done on react to avoid making the tests too long

### CODE REVIEW

File: cypress/integration/react.spec.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  })
+
+  it('Contains Hits', () => {
+    cy.get(HIT_ITEM_CLASS).eq(0).contains('Counter-Strike')
+    cy.get(HIT_ITEM_CLASS).eq(0).contains('9.99 $')
+  })
+
+  it('click on facets', () => {
+    const checkbox = `.ais-RefinementList-list .ais-RefinementList-checkbox`
+    cy.get(`${checkbox}`).eq(1).click()
```
Comment (curquiza) [inline cypress/integration/react.spec.js:47]:
```suggestion
    cy.get(checkbox).eq(1).click()
```

File: cypress/integration/react.spec.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  it('Search', () => {
+    cy.get('.ais-SearchBox-input').type('orwell')
+    cy.wait(1000)
+    cy.get(HIT_ITEM_CLASS).eq(0).contains('Orwell')
+    cy.get(HIT_ITEM_CLASS).eq(0).contains('Late 2016')
+  })
+
+  it('Unclick on facets', () => {
+    const checkbox = `.ais-RefinementList-list .ais-RefinementList-checkbox`
+    cy.get(`${checkbox}`).eq(0).click()
```
Comment (curquiza) [inline cypress/integration/react.spec.js:63]:
```suggestion
    cy.get(checkbox).eq(0).click()
```

File: package.json [hunk 3]
```diff
@@ -7,31 +7,21 @@
...
-    "test:env:esm": "yarn --cwd tests/env/esm && yarn --cwd tests/env/esm start",
-    "test:env:ts": "yarn --cwd tests/env/typescript-node && yarn --cwd tests/env/typescript-node start",
-    "test:playgrounds": "yarn build:playground:vue && yarn build:playground:react && yarn build:playground:javascript && yarn build:playground:html && yarn build:playground:angular",
-    "build:playground:vue": "yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue build",
-    "build:playground:react": "yarn --cwd ./playgrounds/react && yarn --cwd ./playgrounds/react build",
-    "build:playground:javascript": "yarn --cwd ./playgrounds/javascript && yarn --cwd ./playgrounds/javascript build",
-    "build:playground:html": "yarn --cwd ./playgrounds/html && yarn --cwd ./playgrounds/html build",
-    "build:playground:angular": "yarn --cwd ./playgrounds/angular && yarn --cwd ./playgrounds/angular build",
+    "test:e2e": "yarn e2e:angular",
+    "e2e": "sh scripts/e2e.sh",
```
Comment (curquiza) [inline package.json:11]:
```suggestion
    "test:e2e": "sh scripts/e2e.sh",
```

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -10,6 +10,34 @@ on:
       - main
 
 jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    container: cypress/browsers:node12.18.3-chrome87-ff82
```
Comment (curquiza) [inline .github/workflows/test.yml:15]:
```suggestion
    # Only test on Google Chrome
    container: cypress/browsers:node12.18.3-chrome87-ff82
```

**Review by curquiza — CHANGES_REQUESTED** [review]
small changes according to our call 😇

**Review by curquiza — APPROVED** [review]
🏋️

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**